src/internal/modules/charge-information/forms/review.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const Joi = require('@hapi/joi');
+const { formFactory, fields } = require('shared/lib/forms/');
+const routing = require('../lib/routing');
+
+const reviewForm = (request, reviewOutcome, reviewComments) => {
+  const { csrfToken } = request.view;
+
+  const action = routing.postReview(request);
+
+  const f = formFactory(action, 'POST');
+
+  f.fields.push(fields.radio('reviewOutcome', {
+    errors: {
+      'any.required': {
+        message: 'Select yes to approve the charge information or no to request changes.'
+      }
+    },
+    choices: [
+      {
+        value: 'approve',
+        label: 'Yes, approve',
+        fields: [fields.paragraph('approval_notice', {
+          text: 'This licence will be added to the supplementary bill run'
+        })]
+      },
+      {
+        value: 'changes_requested',
+        label: 'No, request a change',
+        fields: [fields.text('reviewerComments', {
+          multiline: true,
+          required: true,
+          errors: {
+            'any.empty': {
+              message: `Enter details into the box about what needs to change.`
+            }
+          },
+          label: 'Which parts of the charge information are incorrect?',
+          hint: 'Describe what needs to change. This will help the person who set up the charge information to correct it.'
+        }, reviewComments)]
+      }
+    ]
+  }, reviewOutcome));
+  f.fields.push(fields.hidden('csrf_token', {}, csrfToken));
+  f.fields.push(fields.button(null, { label: 'Continue' }));
+
+  return f;
+};
+
+const reviewFormSchema = () => ({
+  csrf_token: Joi.string().uuid().required(),
+  approval_notice: Joi.any(),
+  reviewOutcome: Joi.string().required().allow('approve', 'changes_requested'),
+  reviewerComments: Joi.string().when('reviewOutcome', {
+    is: 'changes_requested',
+    then: Joi.string().required(),
+    otherwise: Joi.string().optional().allow('').allow(null).default('')
```

Comment:
I don't think you need this line. See the start date form for charge info. 
src/internal/modules/charge-information/forms/start-date.js

Reply:
I'm afraid this line is necessary, because the reviewerComments field is still there, even if the workflow is `approved`. Leaving it out will result in the error seen below

![Screenshot 2020-10-28 at 16 51 23](https://user-images.githubusercontent.com/6029895/97469011-d3ad9500-193d-11eb-97d2-9f3109499be5.png)

Reply:
This will work

` reviewerComments: Joi.when('reviewOutcome', {
    is: 'changes_requested',
    then: Joi.string().required()`